src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
Specifically the API typing for fetch, I've had a few problems with it in Danger and Peril - but letting it infer has worked fine.